packages/sw-runtime-caching/src/lib/cache-first.js

```diff
@@ -18,7 +18,18 @@ import assert from '../../../../lib/assert';
 
 /**
  * @memberof module:sw-runtime-caching
- * @extends module:sw-runtime-caching.Handler
+ * @extends Handler
```

Comment:
The link doesn't working with the current `documentation`-based generation, no. But it didn't work either with the `module:` prefix, and at least the output is less verbose with that removed.